packages/connect-iframe/src/index.ts

```diff
@@ -177,7 +177,7 @@ const targetUiEvent = (message: CoreMessage) => {
         UI.BUNDLE_PROGRESS,
         UI.ADDRESS_VALIDATION,
     ];
-    return message.event === UI_EVENT && whitelistedMessages.indexOf(message.type) < 0;
+    return message.event === UI_EVENT && blockedMessages.indexOf(message.type) < 0;
```

Comment:
i think that those are the **only** messages that are allowed to be sent to 3rd party window, therefore they are whitelisted

if UI_EVENT message type **is not** whitelisted *[you can change it to .includes(message.type) btw] then the message should be sent to `_popupMessagePort` (for example UI.REQUEST_PIN)

so i think function name should be changed to `shouldUiEventBeSentToPopup` :)

Reply:
`shouldUiEventBeSentToPopup` sounds quite descriptive, I'd do that. We were wrapping our head around these files and I guess we might come up with couple of more improvements in naming that would make them more readable

Reply:
What about this changes ed19cdebcb5066b5cbc25c8e20f3e50df0d51782 ?

Reply:
ok with me